src/callbacks/callbacks.jl

```diff
@@ -102,8 +102,12 @@ NVTX.@annotate function rrtmgp_model_callback!(integrator)
         max_zenith_angle = FT(π) / 2 - eps(FT)
         irradiance = FT(CAP.tot_solar_irrad(params))
         au = FT(CAP.astro_unit(params))
-        (; orbital_data) = p.radiation
-
+        date0 = DateTime("2000-01-01T11:58:56.816")
```

Comment:
Could we use CLIMAParameter for this? (If not in this PR, could you open an issue so that we remember to change this?)

Reply:
Sure.